elements/lisk-p2p/src/p2p.ts

```diff
@@ -997,5 +738,9 @@ export class P2P extends EventEmitter {
 		peerPool.on(EVENT_INBOUND_SOCKET_ERROR, this._handleInboundSocketError);
 		peerPool.on(EVENT_BAN_PEER, this._handleBanPeer);
 	}
+
+	private _bindHandlersToPeerServer(peerServer: PeerServer): void {
+		peerServer.on(EVENT_NEW_INBOUND_PEER, this._handleInboundPeerConnect);
```

Comment:
I think we should also bind `EVENT_FAILED_TO_ADD_INBOUND_PEER` that `peerServer` is emitting as well

Reply:
Seems `EVENT_FAILED_TO_ADD_INBOUND_PEER` could only hold new information if the PeerPool.addInboundPeer() is failing due duplication. 
Any other case EVENT_INBOUND_SOCKET_ERROR is holding all the information.